lib/src/i18n_impl.dart

```diff
@@ -157,17 +157,40 @@ void renderTranslation(Translation translation, StringBuffer output) {
       final className = meta.nest(prefix).objectName.convertName();
       output.writeln('\t$className get $keyName => $className(this);');
     } else {
+      final comment = _wrapWithComments(v);
```

Comment:
Can you move `comment` into separate writeln outside the condition, and remove it from inside the conditional:

```
final comment = _wrapWithComments(v);
output.writeln('\t$comment'); <-- here
if (k.contains('(')) {
```

Reply:
Will do it tonight :)